docs/gatsby-config.js

```diff
@@ -62,17 +66,18 @@ module.exports = {
           Monitoring: ['monitoring/metrics', 'monitoring/health-checks'],
           'API Reference': [
```

Comment:
This section is a little thin now... We don't have good formatting for nesting built-in plugins under API reference, right?  I do think part of solving the middleware-vs-batteries-included puzzle will involve breaking up `api/apollo-server` into multiple files though eventually.

Reply:
Alas we can't go multiple layers down in the nav at this time. Agree this is thin, but at least we can continue to iterate on the nav in the lead-up to release